Sources/UIImageColors/Internal/_ColorAnalyzer.swift

```diff
@@ -0,0 +1,61 @@
+//
+//  _ColorAnalyzer.swift
+//  UIImageColors
+//
+//  Created by Felix Herrmann on 04.10.21.
+//
+
+import Foundation
+
+
+// MARK: - _ColorAnalyzer
+
+internal struct _ColorAnalyzer {
```

Comment:
The underscore prefix is unnecessary. Unless this is idiomatic Swift now?

Reply:
I think it is common to name internal stuff like that, as you can see right [here](https://github.com/apple/swift/blob/3290833eba4d9b34bcc634551f0046722c2cf6ab/stdlib/public/core/Array.swift#L1932)